stoobly_agent/test/config/data_dir_test.py

```diff
@@ -0,0 +1,74 @@
+import os
+import pdb
+import shutil
+
+from stoobly_agent.config.data_dir import DataDir
+
+
+class TestDataDir():
+  original_cwd = os.getcwd()
+
+  def test_in_home(self):
+    home_dir = os.path.expanduser("~")
+    data_dir_path = os.path.join(home_dir, DataDir.DATA_DIR_NAME, 'tmp', DataDir.DATA_DIR_NAME)
```

Comment:
Shouldn't `data_dir_path` just be `os.path.join(home_dir, DataDir.DATA_DIR_NAME)`?

Reply:
The "home dir" in the tests here is `~/.stoobly/tmp/.stoobly`